### PR TITLE
Discover function

### PR DESCRIPTION
**Pull request by VictoriaWika**

### CODE REVIEW

File: src/components/Discover/Discover.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+          likedPlaces={likedPlaces}
+        >
+          {item.name}
+        </AttractionCard>
+      ))}
+    </>
+  )
+  function sightRandomizer() {
+    const AllSightsRandom = AllSights.sort(() => 0.5 - Math.random())
+    let randomSights = AllSightsRandom.slice(0, 5)
```
Comment (badabam) [inline src/components/Discover/Discover.js:29]: No need for `let` here.

File: src/components/Header/Header.js [hunk 2]
```diff
@@ -15,6 +15,7 @@ const AppHeader = styled.header`
   left: 0;
   right: 0;
   padding: 8px;
+  z-index: 30;
```
Comment (badabam) [inline src/components/Header/Header.js:18]: Please consider using css variables to keep track of all used `z-indices`. Could be in `GlobalStyles`.

**Review by badabam — COMMENTED** [review]
Found some small things. Let's talk, if you have any questions.

File: src/index.js [hunk 3]
```diff
@@ -15,7 +15,4 @@ ReactDOM.render(
   document.getElementById('root')
 )
 
-// If you want to start measuring performance in your app, pass a function
```
Comment (BerePendzialek) [inline src/index.js:18]: This reminds me to erase the standart comments in the code ;)

File: src/components/HomePage/HomePage.js [hunk 4]
```diff
@@ -1,19 +1,23 @@
 import styled from 'styled-components/macro'
+import Discover from '../Discover/Discover'
 import Header from '../Header/Header'
 
-export default function HomePage() {
+export default function HomePage({ handleLikePlace, likedPlaces, AllSights }) {
```
Comment (BerePendzialek) [inline src/components/HomePage/HomePage.js:5]: As I read AllSights in capital letter, I thought about a component.

**Review by BerePendzialek — COMMENTED** [review]
Great work!

File: src/App.js [hunk 5]
```diff
@@ -1,27 +1,39 @@
...
   return (
-    <AppLayout>
+    <>
       <Switch>
         <Route exact path="/">
-          <HomePage />
+          <HomePage
+            handleLikePlace={handleLikePlace}
+            likedPlaces={likedPlaces}
+            AllSights={AllSights}
```
[the remark this answers is not in the document]
Comment (VictoriaWika) [reply src/App.js:35]: Yes, you're right!

File: src/components/Discover/Discover.js [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+          onLike={handleLikePlace}
+          likedPlaces={likedPlaces}
+        >
+          {item.name}
+        </AttractionCard>
+      ))}
+    </>
+  )
+  function sightRandomizer() {
+    const AllSightsRandom = AllSights.sort(() => 0.5 - Math.random())
```
[the remark this answers is not in the document]
Comment (VictoriaWika) [reply src/components/Discover/Discover.js:28]: 👍🏻

File: src/components/Header/Header.js (hunk 2 above)
Comment (VictoriaWika) [reply src/components/Header/Header.js:18]: Applied variables for both z-index and color! :)

File: src/components/HomePage/HomePage.js (hunk 4 above)
Comment (VictoriaWika) [reply src/components/HomePage/HomePage.js:5]: Yes you are right, already changed it!

File: src/App.js [hunk 7]
```diff
@@ -1,27 +1,39 @@
...
   const [cards, setCards] = useState(loadFromLocal('cards') ?? [])
 
+  let AllSights = cities
+    .map(function (val) {
+      return val.attraction
+    })
+    .reduce(function (pre, cur) {
+      return pre.concat(cur)
+    })
+
```
[the remark this answers is not in the document]
Comment (VictoriaWika) [reply src/App.js:23]: I have googled, but flatMap was never suggested... But I tried it out and it works perfectly, thanks! 👍🏻

File: src/components/Discover/Discover.js (hunk 1 above)
Comment (VictoriaWika) [reply src/components/Discover/Discover.js:29]: Thanks! 👍🏻

File: src/App.js [hunk 8]
```diff
@@ -1,27 +1,39 @@
...
   }, [cards])
 
   return (
-    <AppLayout>
+    <>
       <Switch>
         <Route exact path="/">
-          <HomePage />
+          <HomePage
+            handleLikePlace={handleLikePlace}
```
[the remark this answers is not in the document]
Comment (VictoriaWika) [reply src/App.js]: Thank you! Missed that one!

**Pull request merged**